docs/Concepts/Pruning.md

```diff
@@ -3,11 +3,11 @@ description: Pruning
 
 # Pruning
 
-Use pruning to reduce storage required for the chain. As the number of transactions increases, the world state grows. Pruning removes state tries that aren't required.
+Use pruning to reduce storage required for the world state size. Pruning removes state trie nodes that aren't required.
```

Comment:
Same comment as the other PR about `size`

Reply:
Removed the term.